src/EditorFeatures/Test2/NavigationBar/TestHelpers.vb

```diff
@@ -148,32 +147,25 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.NavigationBar
 
                 AssertEqual(expectedItem, actualItem, isCaseSensitive)
             Next
-
-            ' Ensure all the actual items that have navigation are distinct
-            Dim navigableItems = actualItems.Select(Function(i) DirectCast(i, WrappedNavigationBarItem).UnderlyingItem).
-                                             OfType(Of RoslynNavigationBarItem.SymbolItem).
-                                             ToList()
-
-            Assert.True(navigableItems.Count() = navigableItems.Distinct(New NavigationBarItemNavigationSymbolComparer(isCaseSensitive)).Count(), "The items were not unique by SymbolID and index.")
```

Comment:
will see about restoring some sort of check here.  but this part was all based on symbolkeys which just has no meaning now.